strongbox-commons/src/main/java/org/carlspring/strongbox/artifact/coordinates/AbstractArtifactCoordinates.java

```diff
@@ -23,13 +23,19 @@ public AbstractArtifactCoordinates()
     public AbstractArtifactCoordinates(Map<String, String> coordinates)
     {
         this.coordinates = coordinates;
+        this.uuid = toPath();
```

Comment:
So... what's the purpose of this...?

Reply:
the purpose that `Path` is unique and can be used as `uuid`

Reply:
The path is unique to the repository. I'm not sure, if this is relevant... (Just thought of mentioning it, just in case).

Reply:
yes, you are right.
seems that i missed that